lib/index.js

```diff
@@ -2,5 +2,6 @@ const requireindex = require('requireindex');
 
 module.exports.rules = requireindex(__dirname + '/rules');
 module.exports.configs = {
-  recommended: require('../config/recommended')
+  recommended: require('../config/recommended'),
```

Comment:
I was going to submit a separate PR for this and batch it with a change in the `no-global-jquery` rule @rwjblue noticed (in the rule we are only checking for global `$` and not also `jQuery`.